src/state/slices/opportunitiesSlice/selectors/stakingSelectors.ts

```diff
@@ -541,11 +540,19 @@ export const selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty =
 
         return true
       })
-      const getTotalProviderBalance = (opportunity: StakingEarnOpportunityType) =>
-        bnOrZero(opportunity.fiatAmount).toNumber()
-      const getApy = (opportunity: StakingEarnOpportunityType) =>
-        bnOrZero(opportunity.apy).toNumber()
-      return orderBy(results, [getTotalProviderBalance, getApy], ['desc', 'desc'])
+
+      const sortedResultsByFiatAmount = results.sort((a, b) =>
+        bnOrZero(a.fiatAmount).gte(bnOrZero(b.fiatAmount)) ? -1 : 1,
+      )
+
+      const activeResults = sortedResultsByFiatAmount.filter(opportunity =>
+        bnOrZero(opportunity.fiatAmount).gt(0),
+      )
+      const inactiveResults = sortedResultsByFiatAmount
+        .filter(opportunity => bnOrZero(opportunity.fiatAmount).eq(0))
```

Comment:
for all of these active vs inactive we can partition if we want https://lodash.com/docs/4.17.15#partition

Reply:
6c33866aa2b84f4fbd026cdcb8a0f880b7a2c076